src/main/resources/plugin.yml

```diff
@@ -1,6 +1,6 @@
 name: RageMode
-main: hu.montlikadani.RageMode.RageMode
-version: 1.0.0
+main: hu.montlikadani.ragemode.RageMode
+version: ${project.version}
```

Comment:
No. I will create a version checker which check this version from GitHub plugin.yml.